hypertrace-view-generator/hypertrace-view-generator/src/test/resources/configs/view-gen-log-event/application.conf

```diff
@@ -0,0 +1,39 @@
+service.name = view-gen-raw-service
+service.admin.port = 8099
+
+main.class = org.hypertrace.core.viewgenerator.service.ViewGeneratorLauncher
+
+input.topic = "raw-logs"
+output.topic = "log-view-events"
+input.class = org.hypertrace.core.datamodel.RawLogEvents
```

Comment:
Is the `org.hypertrace.core.datamodel.RawLogEvents` correct? I think, it should be `org.hypertrace.core.datamodel.LogEvents`?